static/defaults.config.php

```diff
@@ -374,9 +374,13 @@
 		// When the global community page is enabled, use this option to display a hint above the stream, that this is a collection of all public top-level postings that arrive on your node.
 		'show_global_community_hint' => false,
 
-		// show_unsupported_addons (Boolean)
-		// Show all addons including the unsupported ones.
-		'show_unsupported_addons' => false,
```

Comment:
What about this removed config value?

Reply:
Ups ... that had been a cut and paste victim.

Reply:
Can you make a small PR to add this again? I'm busy until Sunday night from exactly now on.

Reply:
Ha ha, ok :)

Reply:
Done: https://github.com/friendica/friendica/commit/22e7cab6d49f5916db5313066167a7a58f30e949